src/theory/strings/sequences_rewriter.cpp

```diff
@@ -1527,7 +1526,7 @@ Node SequencesRewriter::rewriteSeqNth(Node node)
     if (pos < len)
     {
       std::vector<Node> elements = s.getConst<Sequence>().getVec();
-      ret = elements[pos];
+      const Node& ret = elements[pos];
```

Comment:
should the reference be dropped?

Reply:
`const Node` would entail copying from `elements[pos]` to `ret`. This avoids the copy.

Reply:
Yes, and we should avoid copy whenever possible.